maidr/util/environment.py

```diff
@@ -11,3 +15,66 @@ def is_interactive_shell() -> bool:
             )
         except ImportError:
             return False
+
+    @staticmethod
+    def initialize_llm_secrets() -> str:
+        """Inject the LLM API keys into the MAIDR instance."""
+
+        gemini_api_key = os.getenv("GEMINI_API_KEY")
+        openai_api_key = os.getenv("OPENAI_API_KEY")
+
+        # Default settings for the MAIDR instance
+        settings = {
+            "vol": "0.5",
+            "autoPlayRate": "500",
+            "brailleDisplayLength": "32",
+            "colorSelected": "#03c809",
+            "MIN_FREQUENCY": "200",
+            "MAX_FREQUENCY": "1000",
+            "keypressInterval": "2000",
+            "ariaMode": "assertive",
+            "openAIAuthKey": "",
+            "geminiAuthKey": "",
+            "skillLevel": "basic",
+            "skillLevelOther": "",
+            "LLMModel": "openai",
+            "LLMPreferences": "",
+            "LLMOpenAiMulti": False,
+            "LLMGeminiMulti": False,
+            "autoInitLLM": True,
+        }
+
+        if gemini_api_key is not None and openai_api_key is not None:
+            settings["geminiAuthKey"] = gemini_api_key
+            settings["openAIAuthKey"] = openai_api_key
+            settings["LLMOpenAiMulti"] = True
+            settings["LLMGeminiMulti"] = True
+            settings["LLMModel"] = "multi"
+        elif openai_api_key is not None:
+            settings["LLMOpenAiMulti"] = True
+            settings["openAIAuthKey"] = openai_api_key
+        elif gemini_api_key is not None:
+            settings["LLMGeminiMulti"] = True
+            settings["geminiAuthKey"] = gemini_api_key
```

Comment:
Only when gemini key is available, you need to set to gemini. If we don't this, this will fall back to the default "openai" model.

```suggestion
        elif gemini_api_key is not None:
            settings["LLMGeminiMulti"] = True
            settings["geminiAuthKey"] = gemini_api_key
            settings["LLMModel"] = "gemini"
```

Reply:
Thank you Professor @jooyoungseo, will check this!